components/shared/header.tsx

```
@@ -163,74 +166,105 @@ export function Header() {
+                    <>
+                      <button
+                        className={`flex items-center gap-1 transition-colors ${
+                          isActiveDropdown(item.items || [])
+                            ? activeClass
+                            : inactiveClass
+                        }`}
+                        onClick={() => toggleDropdown(item.name)}
+                        onMouseEnter={() => handleMouseEnter(item.name)}
+                        aria-expanded={openDropdowns[item.name]}
+                        aria-haspopup="true"
+                      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)